### PR TITLE
Fix aiohttp client

### PR DESCRIPTION
**Pull request by alxpy**
Fix for broken `aiohttp.client` tracker.

`url` may be not a string:
https://github.com/aio-libs/aiohttp/blob/master/aiohttp/client.py#L1083
https://github.com/aio-libs/aiohttp/blob/master/aiohttp/typedefs.py#L45

### CODE REVIEW

**Comment by beniwohli** [comment]
@alxpy great catch, thanks for opening the PR! If you're up for it, could you write a test for this scenario? You could copy https://github.com/elastic/apm-agent-python/blob/6598165320d6891aa50bb6b178a6a06043efb04a/tests/instrumentation/asyncio/aiohttp_client_tests.py#L41-L59 and turn the `waiting_httpserver.url` into a yarl `URL` object before passing it to `session.get()`. Then, you can use this command to run the tests

    ./tests/scripts/docker/run_tests.sh python-3.8 aiohttp-newest

I can also add the test myself before merging if you prefer :)

**Comment by alxpy** [comment]
I'll write test)

**Comment by alxpy** [comment]
@beniwohli done

File: tests/instrumentation/asyncio/aiohttp_client_tests.py [hunk 1]
```diff
@@ -38,10 +39,14 @@
 pytestmark = [pytest.mark.asyncio, pytest.mark.aiohttp]
 
 
-async def test_http_get(instrument, event_loop, elasticapm_client, waiting_httpserver):
+@pytest.mark.parametrize("use_yarl", [True, False])
```
Comment (beniwohli) [inline tests/instrumentation/asyncio/aiohttp_client_tests.py]: Great use of `parametrize`, much better than my suggestion of copying the test :+1:

File: tests/instrumentation/asyncio/aiohttp_client_tests.py [hunk 2]
```diff
@@ -29,6 +29,7 @@
 #  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 import pytest  # isort:skip
+import yarl
```
Comment (beniwohli) [inline tests/instrumentation/asyncio/aiohttp_client_tests.py]: @alxpy we need to move this import below the `pytest.importorskip` line. `yarl` is only available if aiohttp is installed. This is a side effect of how our test matrix build works. It only installs the requirements needed for a certain test module, and we use `importskip` to gate that.

Only noticed this after submitting the review, sorry!

Comment (alxpy) [reply tests/instrumentation/asyncio/aiohttp_client_tests.py]: no problem, done)

**Comment by beniwohli** [comment]
@alxpy Awesome, thanks!

**Pull request merged**